pkg/bff/status_test.go

```diff
@@ -63,9 +63,7 @@ func (s *bffTestSuite) TestStatus() {
 	require.Equal(s.mainnet.Calls["Status"], 2)
 
 	// Test Status with TestNet Error
-	s.testnet.OnStatus = func(ctx context.Context, in *gds.HealthCheck) (*gds.ServiceState, error) {
-		return nil, status.Error(codes.Unavailable, "unreachable host")
-	}
+	s.testnet.UseError(mock.StatusRPC, codes.Unavailable, "unreachable host")
```

Comment:
Here is an example of `UseError` - I will use `UseFixture` in the Lookup endpoint tests.